02-how-to-use.Rmd

```diff
@@ -2,179 +2,95 @@
+We wrote this book assuming you're at the start of your journey of learning R and of using data science in your education job. The book takes you from installing R to practicing more advanced data science skills like text analysis.
+
+If you've never written a line of R code, we welcome you to the community! We wrote this book for you. Consider reading the book cover to cover and doing all the analysis walkthroughs. We designed the walkthroughs to help give a high-level view of an analysis from start to finish. In our experience, beginning to learn R can be hard, in part because of trying to figure out where to begin. It is a challenge to formulate a goal or path to get there without first understanding what data analysis can do. We wrote the walkthroughs because it is sometimes easier to learn with a concrete example in mind, rather than to try to learn the concepts and principles in the abstract.
+
+Remember that you'll get more benefit from a few minutes of practice every day than you will from long hours of practice every once in a while. Typing code every day, even if it doesn't always run, is a daily practice that invites learning and "a-ha" moments. We know how easy it is to avoid coding when it doesn't feel successful (we've been there), so we've designed this book to deliver frequent small wins to keep the momentum going. But even so, we all eventually hit a wall in our learning. When that happens, take a break and then come back and
 keep coding. When daily coding becomes a habit, so does the learning.
 
-If you get stuck in an advanced chapter and you need a break, try reviewing an
-earlier chapter. You'll be surprised at how much you learn from reviewing old
-material with the benefit of new experiences. Sometimes that kind of back-to-basics
+If you get stuck in an advanced chapter and you need a break, try reviewing an earlier chapter. You'll be surprised at how much you learn from reviewing old material with the benefit of new experiences. Sometimes that kind of back-to-basics
 attitude is what we need to get a fresh perspective on new challenges.
 
 ### Pick a chapter of interest and start there
 
-We interviewed R users in education as research for this book. We chose people
-with different levels of experience in R, in the education field, and in statistics.
-We asked each interviewee to rate their level of experience on a scale from 1
-to 5, with 1 being "no experience" and 5 being "very experienced". You can try this
-now---take a moment to rate your level of experience in:
+We interviewed R users in education as research for this book. We chose people with different levels of experience in R, in the education field, and in statistics.
+We asked each interviewee to rate their level of experience on a scale from 1 to 5, with 1 being "no experience" and 5 being "very experienced". You can try this now---take a moment to rate your level of experience in:
```

Comment:
I think it would be fun (and interesting) to very briefly describe what we found - were there consistencies in what the people we interviewed said? 

Related, this paper speaks to this a bit: https://link.springer.com/article/10.1007/s41686-023-00083-7

As does another that is on... interviews with educational data scientists! Maybe we can discuss whether we can find any data from the interviews we mention in this text, and if not, I can cite one or both of these papers.

Reply:
We could also cite this, which is an earlier version of the interview piece: https://repository.isls.org/handle/1/7394

Reply:
@jrosen48 do you remember where the interview notes are?

Reply:
@jrosen48 would you mind adding this since you're familiar with the interviews?

Reply:
Ah sorry, we are talking about two different interviews:

1. The interviews we did for the book (I didn't do any of these!)
2. Interviews for that paper (I did those!)